web/client/components/widgets/builder/wizard/map/enhancers/handleMapSelect.js

```diff
@@ -11,8 +11,8 @@ import { compose, withState, mapPropsStream, withHandlers } from 'recompose';
 import GeoStoreDAO from '../../../../../../api/GeoStoreDAO';
 import axios from '../../../../../../libs/ajax';
 import ConfigUtils from '../../../../../../utils/ConfigUtils';
-import { excludeGoogleBackground } from '../../../../../../utils/LayersUtils';
-
+import { excludeGoogleBackground, extractTileMatrixFromSources } from '../../../../../../utils/LayersUtils';
```

Comment:
```suggestion
import { excludeGoogleBackground, extractTileMatrixFromSources } from '../../../../../../utils/LayersUtils';
import '../../../../../../libs/bindings/rxjsRecompose';
```
This dependency is required to use components that needs rxjs + recompose (expecially for testing them alone)